triveni-codev/GBWebApp-dev
# Darlp networks caption settings 1203376941237109

Hi @triveni-codev @chriscodev @pauljustinhernandez 

Kindly check this PR. This is only for Default Service Caption Settings tab under Services only for ATSC Terrestrial type

ASANA: https://app.asana.com/0/1201943683850147/1203376941237109

To test:

-should have the same set and order of languages from Java App
-should be able to retrieve service caption settings from Java App and add/update them
-should be able to add one or more service caption settings
-should be able to update one or more service caption settings
-cancellation should be working fine including all upper modals: networks, channels, services and service caption settings modal
-test Big Commit for Adding (Updating is still under development)

Don't test Big Commit for Updating yet.
